parse.y

```diff
@@ -3830,7 +3830,7 @@ f_larglist	: '(' f_args opt_bv_decl ')'
 			$$ = $2;
 			p->max_numparam = ORDINAL_PARAM;
 		    /*% %*/
-		    /*% ripper: paren!($2) %*/
+		    /*% ripper: lambda_var!(paren!($2), escape_Qundef($3)) %*/
```

Comment:
@mame and I discussed now, and we think that `lambda_var` would be better inside `paren`.
```suggestion
		    /*% ripper: paren!(lambda_var!($2, escape_Qundef($3))) %*/
```

Reply:
Sounds good!